simple-storage/src/main/AndroidManifest.xml

```diff
@@ -1 +1,5 @@
-<manifest package="com.sromku.simple.storage"/>
+<manifest package="com.sromku.simple.storage"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
no need for manifest changes. in fact, I will delete the app here, and leave the lib only.